modules/virt-supported-features-for-virtual-storage.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-supported-features-for-virtual-storage.adoc
+
+[id="virt-supported-features-for-virtual-storage_{context}"]
+= Supported features for {VirtProductName} virtual storage
+
+.{VirtProductName} Virtual storage feature support matrix
+[cols="40%,15%,15%,15%,15%"]
+|===
+||Virtual machine live migration  |Host-assisted virtual machine disk cloning |Storage-assisted virtual machine disk cloning |Virtual machine snapshots
+
+|{product-title} container storage: RBD block volumes
+|Supported
+|Supported
+|Supported
+|Supported
+
+|{product-title} container storage: RBD filesystem volumes
+|Not supported
+|Supported
+|Supported
+|Supported
+
+|{VirtProductName} hostpath provisioner
```

Comment:
@bgaydosrh Should {product-title} and {VirtProductName} have definite names, or they are various depends on what users have?

Reply:
Hi @qwang1 - yes, we use variables for the product names, so in case those names change the doc will always have the most current names.

Those names should appear correctly in the next test build.